app/controllers/topics_controller.rb

```diff
@@ -55,7 +71,7 @@ def scope
     @scope ||= if Current.user.is_admin?
       Topic.all
     else
-      Topic.active
-    end
+      Current.user.topics
```

Comment:
Since we should filter by state, all topics are available for admins.
But for regular user only topics from his providers are available